<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorAtemAudioGainWidget</class>
 <widget class="QWidget" name="InspectorAtemAudioGainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>112</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutInput">
     <item>
      <spacer name="horizontalSpacerInput">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelInput">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutGain">
     <item>
      <spacer name="horizontalSpacerGain">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>64</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelGain">
       <property name="text">
        <string>Gain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderGain">
       <property name="minimum">
        <number>-6000</number>
       </property>
       <property name="maximum">
        <number>600</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxGain">
       <property name="minimum">
        <double>-60.000000000000000</double>
       </property>
       <property name="maximum">
        <double>6.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTriggerOnNext">
     <item>
      <spacer name="horizontalSpacerTriggerOnNext1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTriggerOnNext">
       <property name="text">
        <string>Trigger on next</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTriggerOnNext">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerTriggerOnNext2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerInput">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxTriggerOnNext</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorAtemAudioGainWidget</receiver>
   <slot>triggerOnNextChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderGain</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorAtemAudioGainWidget</receiver>
   <slot>sliderGainChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxGain</sender>
   <signal>valueChanged(double)</signal>
   <receiver>InspectorAtemAudioGainWidget</receiver>
   <slot>doubleSpinBoxGainChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxInput</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>InspectorAtemAudioGainWidget</receiver>
   <slot>inputChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputChanged(int)</slot>
  <slot>sliderGainChanged(int)</slot>
  <slot>triggerOnNextChanged(int)</slot>
  <slot>doubleSpinBoxGainChanged(double)</slot>
 </slots>
</ui>
